### PR TITLE
Spike gt

### PR DESCRIPTION
**Pull request by Gabe-Torres**
Created stat tracker spec file 
Required files in spec helper
Renamed files in futbol

### CODE REVIEW

**Review by Dav1s-Ops — CHANGES_REQUESTED** [review]
Looks good. Since we changed the file names, let's update the file path names as well in the spec_helper and stat_tracker_spec.

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -1,2 +1,7 @@
 require 'simplecov'
+require './lib/stat_tracker'
+require './lib/games'
+require './lib/game_teams'
+require './lib/teams'
+
 SimpleCov.start
```
Comment (Dav1s-Ops) [inline spec/spec_helper.rb]: SimpleCov.start should move up to rb:1.5 (line 2)

**Review by Dav1s-Ops — APPROVED** [review]
Awesome! Looks grand!

**Review by kcockett — APPROVED** [review]
Looks good.

File: spec/stat_tracker_spec.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
+require './spec/spec_helper'
+
+
```
Comment (WZale) [inline spec/stat_tracker_spec.rb:3]: This looks like a great start for our RSpec tests!

**Comment by WZale** [comment]
Good call on renaming the class files, too. I'm glad you caught my mistake there, and that we'll be sticking to Ruby convention!

**Pull request merged**